doc/ref_arch/openstack/chapters/chapter01.md

```diff
@@ -55,6 +56,37 @@ OpenStack considers the following Four Opens essential for success:
 
 This OpenStack Reference Architecture is organised around the three major NFVI resource types as core services of compute, storage and networking, and a set of shared services of identity management, image management, graphical user interface, orchestration engine, etc. 
 
+<a name="1.4.1"></a>
+### 1.4.1 Exceptions
+
+CNTT specifies certain policies and [principles](https://github.com/cntt-n/CNTT/blob/master/doc/common/chapter00.md#2.0) but technology, performance and other considerations makes conformance to these primciples and policies impractical. CNTT specifies how to handle [non-conforming technologies](https://github.com/cntt-n/CNTT/blob/master/doc/common/policies.md#cntt-policies-for-managing-non-conforming-technologies). In general, non-coformance with policies is handled through a set of exceptions (please also see [Exception Types](https://github.com/cntt-n/CNTT/blob/master/doc/common/policies.md#cntt-policies-for-managing-non-conforming-technologies)). 
+
+The following sub-sections list the exceptions to the CNTT principles and shall be updated whenever technology choices, versions and requirements change. The Exceptions have an associted period of validity and this period shall include time for transitioning.
+
+#### 1.4.1.1 Technology Exceptions
+
+The list of Technology Exceptions will be updated or removed when alternative technologies aligned with CNTT principles develop and mature.
+
+| Ref |	Type |	Name |	Description | Valid Until |
+|-----|------|------|-------------|-------------|
+| ra1.exc.tec.001 |	Technology |	SR-IOV |	This exception allows workloads to use SR-IOV over PCI-PassThrough technology. | TBD |
+
+#### 1.4.1.2 Version Exceptions
```

Comment:
I'm not sure I understand this section - what is the exception "for"?

Reply:
@tomkivlin Version Exceptions are for software versions, API versions, etc. For example, while we are using OpenStack Train, for a given service we could limit use of API to an earlier version. The full Cyborg API v2 is available on Ussuri and only partial set on Train and so we could specify that if using Cyborg utilise only API v1 not API v2.

Reply:
Ah ok understood.